packages/meditrak-server/src/apiV2/postChanges.js

```diff
@@ -182,8 +182,10 @@ const requiresSurveyResponseTranslation = (
   const answerTranslatorFields = Object.keys(answerTranslators);
   const { answers } = surveyResponseObject;
   if (Array.isArray(answers)) {
-    return answers.some(answer =>
-      Object.keys(answer).some(field => answerTranslatorFields.includes(field)),
+    return answers.some(
+      answer =>
+        answer.body === '' ||
```

Comment:
So this is for just the empty string case? As the undefined/null should just be omitted from the payload by JSON?

Reply:
At this point the app will have translated anything like that into string format, so we wouldn't expect to see a body of any other type, and if we ever do, I'd like it to still error so we investigate. 

The awkward bit is that if there was a `null` response that the same bug in the app is failing to filter out, by this point on the server it would be `'null'`, but that's indistinguishable from a user actually typing in 'null' and submitting that, so we just can't handle that case. It shouldn't matter too much if there are some `'null'` answers in the db, and I will try to get the app update deployed asap so the window of them potentially making it through is short.